code-challenges.test.js

```diff
@@ -11,31 +10,69 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const fibLength1 = 6
+// input: any number greater than 2
+// expected output: the length of the fibonaci sequence at the index of the number inputed. I want to create a function that will take a number and give me the fibonacci sequence up to that index.
+
+// // a) Create a test with expect statements for each of the variables provided.
+describe("fibLength", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(fibLength(fibLength1)).toEqual([1, 1, 2, 3, 5, 8]);
+    expect(fibLength(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  });
+});
+// got the jest to work! now just need to make the function that will make this pass.
+const fibLength1 = 6;
 // Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
+const fibLength2 = 10;
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// i made an array holding the fib sequence
+let fib = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55];
+// made the function that will take a number as an argument and return the length of the fib sequence at the number of the index given
+const fibLength = (array) => {
+  let newFib = [];
+  for (let i = 0; i < i[array]; i++) {
+    newFib = map.length(array);
```

Comment:
Some things to consider is first, we are being passed a number which will help us determine the length of the array we are creating.  Since we have that number, some modifications on the for loop to take that into consideration could help.  I also see on line 34 that you are assigning newFib to map.length however I don't see map defined anywhere.